spinalcordtoolbox/utils.py

```diff
@@ -430,6 +453,46 @@ def sct_progress_bar(*args, **kwargs):
     return tqdm.tqdm(*args, **kwargs)
 
 
+def zipdir(path_to_zip, fname_zip='archive.zip'):
```

Comment:
I would argue that this function should not be in utils, because it's only needed by one piece of code.

Reply:
i can imagine a developer in the future who might need this function, and will (hopefully) look at our (soon-to-be) API documentation. Zipping a folder is a fairly common task, that could be needed somewhere else, no?